File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typerCarAccepted = []
    seatsMaterial    = []

    def __init__(self, licence, driver, typeCarAccepted, seatsMaterial):
        super().__init__(licence, driver)
        self.typerCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: Java/UberBlack.java
package Java;
import java.util.Map;
import java.util.ArrayList;


public class UberBlack extends Car{

    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String licence, Account driver, Map<String, Map<String, Integer>> typeCarAccepted, ArrayList seatsMaterial){
        super(licence, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}
